Remove a useless instruction

Hywan opened the pull request:
The `text` variable is first initialized to `this.placeholder`, which is strictly equivalent to use `self.placeholder` inside the function used just after. So in case `selected.length` is falsy, this is not necessary to override `text` to the same value, i.e. its inital value.

Also fix a CS.

Hywan commented:
Build is broken because of an HTTP issue when setting up the environment.

apsdehal commented:
@Hywan Are you willing to update your PR according to latest master?

Hywan commented:
`master` no longer has the issue.

apsdehal commented:
nope, it still has it https://github.com/jquery/jquery-mobile/blob/master/js/widgets/forms/select.js#L259

Hywan committed:
Remove a useless instruction.

The `text` variable is first initialized to `this.placeholder`, which is
strictly equivalent to use `that.placeholder` inside the function used
just after. So in case `selected.length` is falsy, this is not necessary
to override `text` to the same value, i.e. its inital value.

Hywan commented:
@apsdehal Done :smiley:.

apsdehal commented:
@Hywan This has landed into master 2f00774. Thanks a lot.

Hywan commented:
Thanks :-).
